sfputil/debug.py

```diff
@@ -112,6 +114,39 @@ def set_output(port_name, enable, direction):
 
         time.sleep(TX_RX_OUTPUT_UPDATE_WAIT_TIME)
 
+        disable_status_list = get_disable_status_func()
+        if not disable_status_list:
+            click.echo(f"{port_name}: {direction.upper()} disable status not available for subport {subport}")
+            sys.exit(EXIT_FAIL)
+
+        for lane in range(1, CMIS_MAX_CHANNELS + 1):
+            if lane_mask & (1 << (lane - 1)):
+                disable_status = disable_status_list[lane - 1]
+                if enable == "disable":
+                    if not disable_status or disable_status == "N/A":
+                        click.echo(f"{port_name}: {direction.upper()} output on lane {lane} is still "
+                                   f"enabled on subport {subport}. Restoring state.")
+                        sys.exit(EXIT_FAIL)
+                else:
+                    if disable_status or disable_status == "N/A":
+                        click.echo(f"{port_name}: {direction.upper()} output on lane {lane} is still "
+                                   f"disabled on subport {subport}. Restoring state.")
+                        sys.exit(EXIT_FAIL)
+
+        if hasattr(api, 'get_cmis_rev'):
+            cmis_rev = api.get_cmis_rev()
+            if cmis_rev is None:
+                click.echo(f"{port_name}: CMIS revision not available for subport {subport}")
+                sys.exit(EXIT_FAIL)
+
+            # OutputStatusRx and OutputStatusTx are supported from CMIS 5.0
+            if float(cmis_rev) < 5.0:
+                click.echo(
+                    f"{port_name}: {direction.upper()} output "
```

Comment:
@Junchao-Mellanox user is not clear why failed from the log message

Reply:
Hi @prgeor , this is not a failure but an expected branch when cmis version is lower than 5.0. You can see I did not return error here.

Reply:
@Junchao-Mellanox Why do we not check for `OutputStatusRx` and `OutputStatusTx` for CMIS version < 5.0? I know its not available in < 5.0. Checking only Tx disable bits does not mean the module has muted the output

Reply:
Hi @prgeor , I am not sure I understand. `OutputStatusRx` and `OutputStatusTx` are not supported when CMIS < 5.0, how can we check it?